baseline/client/tests/verbal-learning.test.js

```diff
@@ -4,15 +4,28 @@ import { clickContinue } from "./utils.js";
 
 describe("verbal-learning", () => {
     it("results should have at least one result marked isRelevant", () => {
-        const timeline = (new VerbalLearning()).getTimeline();
-        let hasRelevantData = false;
-        for (const task of timeline) {
-            if (task.data && task.data.isRelevant) {
-                hasRelevantData = true;
-                break;
+        const timeline = [
+            {timeline: new VerbalLearning(1, 1, () => 0).getTimeline()},
+            {timeline: new VerbalLearning(1, 2, () => Date.now()).getTimeline()},
+            {timeline: new VerbalLearning(1, 3, () => Date.now()).getTimeline()},
+        ];
+        const timelineHasRelevantData = timeline => {
```

Comment:
Rewritten because previous test didn't expect the timeline to contain sub-timelines.